src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/SNSService.kt

```diff
@@ -112,3 +114,35 @@ class SNSAppointmentService(
     }
   }
 }
+
+@Service
+class SNSAppointmentService(
```

Comment:
it's a real shame we can't combine these events - i guess the reason is that we can't generate a link to action plan appointments without the session?